core/src/Revolution/Processors/Browser/File/Download.php

```diff
@@ -45,7 +45,6 @@ public function process()
 
         // Download file
         @session_write_close();
-        $file = $this->sanitize($this->getProperty('file'));
```

Comment:
Were you having issues with this applied? Seems like a good idea to not blankly accept whatever file parameter is provided, and the sanitize method probably helps with that.

Reply:
@Mark-H it's a double call of the `sanitize` method. It's already called on line 36.

Reply:
Ahh, gotcha 👍